test/test-functions

```diff
@@ -17,6 +17,15 @@ UNIFIED_CGROUP_HIERARCHY="${UNIFIED_CGROUP_HIERARCHY:-default}"
 EFI_MOUNT="$(bootctl -x 2>/dev/null || echo /boot)"
 QEMU_MEM="${QEMU_MEM:-512M}"
 
+# Decide if we can (and want to) run QEMU with KVM acceleration.
+if [[ -z "$TEST_NESTED_KVM" ]]; then
```

Comment:
How about using a single check?

```sh
if [[ -n "$TEST_NESTED_KVM" || ( -z "$TEST_NO_KVM" && $(systemd-detect-virt -v) != kvm ) ]]; then
    QEMU_KVM=yes
else
    QEMU_KVM=no
fi
```

Comment could spell out everything quite explicitly:

```
# If nested KVM is requested explicitly, enable it.
# Otherwise, check if it's been disabled explicitly.
# Or fall back to checking whether we're already
# running inside KVM, in which case disable it to
# avoid issues with nested KVM.
```

Or something similar to that...

What do you think?

Reply:
Thanks! I was afraid ending up with a single oneliner with a huge, indecipherable comment, but this is looks nice.